source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -0,0 +1,185 @@
+===========================
+Update Arrays in a Document
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to update array elements in a document.
+
+To update elements in an array, perform the following actions:
+
+- Provide an :ref:`update document <update-document>` specifying the update *(not discussed in this guide)*
```

Comment:
couple of notes:

- I think you can remove *(not discussed in this guide)*. This is implied by the link to another page.
- End each of these list items with a period (per style guide: https://docs.mongodb.com/meta/style-guide/style/lists/#list-items)
- clarify the gerund per https://docs.mongodb.com/meta/style-guide/writing/clarify-gerunds-participles/
```suggestion
- Provide an :ref:`update document <update-document>` that specifies the update.
```